src/pages/en/developers/apis/6-terms-of-use/index.md

```diff
@@ -2,6 +2,74 @@
 
-Route geometries from routing API, address data from geocoding API and map data from map API are licensed under the [Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/index.html).
+### 0. Intro on Digitransit  
+
+Digitransit Platform is an open-source journey planning solution that combines open-source components into a route planning service. It has Mobile friendly user interface, Map tile serving, Geocoding, and data conversion tools. Route planning algorithms and APIs are provided by Open Trip Planner (OTP). More on Digitransit, please visit: https://digitransit.fi/en/
+
+Digitransit is provided by Helsinki Region Transportation (HSL), Fintraffic, and TVV LMJ Oy.
+
+### 1. General  
+
+Digitransit is striving for transparent co-operation with parties who need and want to take advantage of Digitransit data and enables the use of data to foster free and innovative use cases. Digitransit data is guided by European Parliament directive (EU) 2019/1024 and open data and the reuse of public-sector information “Laki julkisen hallinnon tiedonhallinnasta annetun lain muuttamisesta (710/2021)”.
+
+To fulfill this objective Digitransit has opened API-interface for system developers (Users). User may for example, integrate other systems to connect with Digitransit API or develop new applications to utilize the data. These terms of use define the practices, responsibilities, and rights on utilizing the Digitransit API. The User joining to connect to the interface accepts these terms including HSL and Digitransit joint privacy statement. Digitransit is not charging any fees on joining and utilizing API.
+
+### 2. The practices to join the API 
+
+API user can register at https://portal-api.digitransit.fi/. In the registration process the User is required to provide contact information for Digitransit for communications purpose. The communications may be on topics of version changes of API, new services, or good practices for example. After the registration process the User can obtain needed subscription key to authorize API queries. The user registration information is processed according to HSL and Digitransit joint privacy statement (see Other references).
+
+The User can terminate the account and its subscriptions in same address at https://portal-api.digitransit.fi/. (Technically in termination process the user account is deleted both in Digitransit API management and Digitransit ID management systems)
+
+### 3. Digitransit data  
+
+Via API the User can access Digitransit data which is defined in https://digitransit.fi/en/developers/apis/.
+
+Usage of API may require a registration and account verification by e-mail. Also, users first and last name are required. Digitransit may change the identification technology, or the data required in registration any time based on Digitransit changing business needs, e.g., information security and the change applies also to Users that have registered earlier.
```

Comment:
Usage of API may require a registration and account verification by e-mail. Also, User's first and last name are required. Digitransit may change the identification technology, or the data required in registration at any time based on Digitransit business needs such as information security.  Such changes apply also to Users that have registered earlier.